.env.example

```diff
@@ -14,3 +15,5 @@ PANTRY_DIRECTORY_ID=
 WAITRESS_DIRECTORY_ID=
 GDRIVE_TOKEN=
```

Comment:
The env variable was already in the codebase. Since I don't have the various tokens. I could not test them. From the code, it is used by the `upload_to_drive` function to upload to GoogleDrive.